test/data/Csaszar.ply

```diff
@@ -1,30 +1,30 @@
-ply
-format ascii 1.0
-element vertex 7
-property float32 x
-property float32 y
-property float32 z
-element face 14
-property list uchar int vertex_indices
-end_header
--20 -20 -10
--20 20 -15
--5 -8 8
-0 0 30
-5 8 8
-20 -20 -15
-20 20 -10
-3    1     3     6
-3    1     6     5
-3    2     5     6
-3    0     2     6
-3    0     6     4
-3    3     4     6
-3    1     2     3
-3    1     4     2
-3    1     0     4
-3    1     5     0
-3    3     5     4
-3    0     5     3
-3    0     3     2
+ply
```

Comment:
These changes are probably not necessary. The test source changes aren't either.